javascript/javascript2/week1/shortword-daletters.js

```diff
@@ -0,0 +1,25 @@
+const danishWords = ["bil", "plante", "kaffe", "bog", "ø", "planetarium"];
+
+function shortWord(array) {
+  return array.reduce(function(prevWord, currWord) {
+    if (currWord.length < prevWord.length) {
+      return currWord;
+    } else
+      return prevWord;
```

Comment:
Nice!
You don't need the `else` here seeing as you've already returned in the if-statement

Reply:
👍🏻